src/gui/utils.ts

```diff
@@ -210,8 +212,8 @@ export const getElementFile = (path: string): object | undefined => {
             updateFilePath("");
         }
         return undefined;
-    } catch (error) {
-        console.info(error);
+    } catch (error: any) {
```

Comment:
any is not great does unknown work here ?

Reply:
there will be an error show by ts in getLog().error() if it is unknown. Any idea on how to fix that?